scripts/ci/parse_bot_pr_comment.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+from __future__ import annotations
+
+import os
+import re
+import sys
+
+"""
+Script for parsing @rerun-bot PR comments.
+
+In order to avoid executing code that an attacker might embed in a comment,
+the comment body is not passed as an argument, but instead through the `GITHUB_COMMENT_BODY` env
+variable which is set by the GitHub Actions workflow.
+
+Writes the desired content of `GITHUB_OUTPUT` to stdout.
+"""
+
+# List of valid inputs for the bot
+valid_inputs = ["full-check"]
+
+
+comment_body = os.environ.get("GITHUB_COMMENT_BODY", "")
+bot_invocation = re.search(r"@rerun-bot\s+([a-zA-Z\-]*)", comment_body)
+if bot_invocation:
+    command = bot_invocation.group(1)
+    if str(command) in valid_inputs:
+        print(f"command={command}")
```

Comment:
I think we should directly write to `GITHUB_OUTPUT` here, using something like this (untested)

```py
def gh_set_output(key: str, text: str) -> None:
    line = f"{key}={text}"
    output_file = os.environ.get("GITHUB_OUTPUT")
    if output_file is not None:
        # append to output_file
        with open(output_file, "a", encoding="utf8") as f:
            f.write(f"{line}{os.linesep}")
    else:
        # print when not running on CI
        print(line)
```

```suggestion
        gh_set_output("command", command)
```

I do a similar thing here for our custom Vercel action:
https://github.com/rerun-io/rerun/blob/ab526f356775ac5fadb4f26742ca215bffe045b8/.github/actions/vercel/util.mjs#L35-L43

Reply:
ah, I wasn't sure if that would be exported all the way, somehow assumed that the commands run sandboxed, but that was a different ci system I worked with at some point
👍